ant-node/src/node.rs

```diff
@@ -580,11 +586,8 @@ impl Node {
         Ok(())
     }
 
-    async fn handle_query(
-        network: &Network,
-        query: Query,
-        payment_address: RewardsAddress,
-    ) -> Response {
+    async fn handle_query(node: Self, query: Query, payment_address: RewardsAddress) -> Response {
+        let network = node.network();
```

Comment:
Shouldn't this become a proper method of node? Maybe `&self` as we don't need ownership here do we?